com.github.Eloston.UngoogledChromium.metainfo.xml

```diff
@@ -7,7 +7,7 @@
   <developer_name>Eloston and community</developer_name>
   <summary>A lightweight approach to removing Google web service dependency</summary>
   <metadata_license>CC0-1.0</metadata_license>
-  <project_license>MPL-2.0</project_license>
+  <project_license>BSD-3-Clause</project_license>
```

Comment:
Can you change it to `  <project_license>BSD-3-Clause and LGPL-2.1+ and Apache-2.0 and IJG and MIT and GPL-2.0+ and ISC and OpenSSL and (MPL-1.1 or GPL-2.0 or LGPL-2.0)</project_license>`. This is what org.chromium.Chromium has set